### PR TITLE
added hello world fixture

### PR DESCRIPTION
**Pull request by Heulitig**

### CODE REVIEW

File: integration-tests/_tests/fixtures/hello-world-fixture.test.ftd [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+-- fastn.get: Fetching Test Data (from test server)
+url: /get-data/
+http-status: 200
+
+-- fastn.get.test:
+
+fastn.assert.eq(fastn.http_response["data"], "Hello, World!");
+
+-- end: fastn.test
```
Comment (amitu) [inline integration-tests/_tests/fixtures/hello-world-fixture.test.ftd:13]: Do we need this end statement? We can change the type of `fastn.test` to just a component without children.

**Comment by amitu** [comment]
@Heulitig the test is failing.

File: integration-tests/_tests/fixtures/hello-world-fixture.test.ftd (hunk 1 above)
Comment (Heulitig) [reply integration-tests/_tests/fixtures/hello-world-fixture.test.ftd]: Removed children from fastn.test component

File: integration-tests/_tests/01-hello-world-using-sql-processor.test.ftd [hunk 2]
```diff
@@ -1,5 +1,7 @@
 -- import: fastn
 
+-- fastn.test: 01-hello-world-using-sql-processor
```
Comment (amitu) [inline integration-tests/_tests/01-hello-world-using-sql-processor.test.ftd:3]: The test name (caption) should be optional.

Comment (amitu) [reply integration-tests/_tests/01-hello-world-using-sql-processor.test.ftd:3]: To prove that, you can leave out caption in one of the tests.

**Review by amitu — APPROVED** [review]

**Pull request merged**